index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="header-and-headline-section">
+        <header>
+                <nav class="navbar">
+                    <a href="#" class="logo">Cobby Elson</a>
+                    <button type="button" class="hamburger-btn">
+                        <span class="toggler-icon top-bar"></span>
+                        <span class="toggler-icon middle-bar"></span>
+                        <span class="toggler-icon bottom-bar"></span>
+                    </button>
+                </nav>
+        </header>
+        <section>
+                <div class="about-section">
+                    <div class="container-for-about-section">
+                        <div class="about-section-item">
+                            <h1 class="intro">
+                                I’m Cobby. Glad to see you!
+                            </h1>
+                        </div>
+                        <div class="about-section-item">
+                            <p class="profile">I’m a software developer! I can help you build a product , 
+                                feature or website Look through some of my work and experience! 
+                                If you like what you see and have a project you need coded, 
+                                don’t hesitate to contact me.
+                            </p>
+                        </div>
+                        <div class="about-section-item links">
+                            <a href="http://" class="lets-connect">LET’S CONNECT</a>
+                            <ul class="social-media-icons">
+                                <li><img class="twitter" src="./assets//twitter.png" alt="twitter"></li>
+                                <li><img class="link" src="./assets//Linkedin icon.png" alt="linkedin"></li>
+                                <li><img class="socialIcon" src="./assets/socialIcon.png" alt="social"></li>
+                                <li><img class="git" src="./assets/git.png" alt="git"></li>
+                                <li><img class="twohandicon" src="./assets/twohandicon.png" alt="my pub"></li>
+                            </ul> 
```

Comment:
- [ ] [optional] i will mark this as optional suggestion as your previouse reviewer didnt state this foryou, please i would suggest you to use `a` tag in all your socail icon  ,according to the project requirement The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag) for proper semantic reason.

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href please fix this .

![Screenshot 2023-06-02 014043](https://github.com/CobbyElsonfx/Portfolio-Website/assets/92458236/d0de6c0c-e8d4-46f9-9273-9e104abd43dc)